.github/workflows/generator-generic-ossf-slsa3-publish.yml

```
@@ -0,0 +1,69 @@
+
+  provenance:
+    needs: [build]
+    permissions:
+      actions: read   # To read the workflow path.
+      id-token: write # To sign the provenance.
+      contents: write # To add assets to a release.
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@v1.4.0
+    with:
+      base64-subjects: "${{ needs.build.outputs.digests }}"
+      upload-assets: true # Optional: Upload to a new release
+      eth:100
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)